docker-compose.yml

```diff
@@ -6,6 +6,7 @@ networks:
 
 services:
   server:
+    image: server
```

Comment:
@perryrobinson can you help explain this line? everything else looks good

where is this name coming from? and when is it refreshed?

Reply:
I was just adding a name to the image that is built by the docker compose up -d --build command. Otherwise docker just gives the created image some random name.

Reply:
thanks for educating me :)

Reply:
> @NxP4Code yes for sure - if that's something which you would be willing to help with, we would happily help shepherd that PR and land it! no pressure though

Yes I created PR for this.

Reply:
thank you!